### PR TITLE
Adding images to supplement the textual description

### PR DESCRIPTION
**Pull request by yashk2810**
I created the images so no need for license :smile:

### CODE REVIEW

File: site/en/tutorials/eager/_pix2pix.ipynb [hunk 1]
```diff
@@ -608,6 +626,15 @@
...
         "* Then, we calculate the gradients of loss with respect to both the generator and the discriminator variables(inputs) and apply those to the optimizer.\n",
+        "* This entire procedure is shown in the images below.\n",
+        "\n",
+        "![alt text](https://raw.githubusercontent.com/yashk2810/yashk2810.github.io/master/dis.jpg)\n",
+        "\n",
+        "\n",
+        "---\n",
+        "\n",
+        "\n",
+        "![alt text](https://raw.githubusercontent.com/yashk2810/yashk2810.github.io/master/gen.jpg)\n",
```
[the remark this answers is not in the document]
Comment (yashk2810) [reply site/en/tutorials/eager/_pix2pix.ipynb]: > We need to host these somewhere else.

Yup, you can download those images and host it on tensorflow's thing.

> Also, diagrams are difficult for us translate

Difficult to translate in what sense? The diagrams just describe the procedure mentioned above in the loss function section and the training section.

> we need the raw file for updates.

Yes, I'll put that up.

If any changes is required for the diagram, please let me know and I'll be happy to do it.

[the remark this answers is not in the document]
Comment (lamberta) [reply site/en/tutorials/eager/_pix2pix.ipynb]: > Also, diagrams are difficult for us translate

As in, diagrams/images there not automatically translated with the text and require special instructions (so they probably won't be translated). Additionally, different languages tend to have different word lengths, so, for example, a German phrase probably won't fit inside the same box that fits an English phrase. Best to place words outside the box.

I don't think we want to rely on draw.io for images. Can you export an SVG?

[the remark this answers is not in the document]
Comment (lamberta) [reply site/en/tutorials/eager/_pix2pix.ipynb]: Sorry, I'm talking about language translation of tensorflow.org content to Chinese, etc.

[the remark this answers is not in the document]
Comment (yashk2810) [reply site/en/tutorials/eager/_pix2pix.ipynb]: So what changes should I do? Taking text out of the boxes won't look nice and clean 😅

[the remark this answers is not in the document]
Comment (yashk2810) [reply site/en/tutorials/eager/_pix2pix.ipynb]: Can we change the images when translating to other languages? For example, if the text goes out of the box, increasing the size of the box.

**Comment by yashk2810** [comment]
@MarkDaoust On a separate note, is it a good idea to shift the print statements to use            f-strings instead of .format()?

This would be a great start to scrub out python 2 related syntax. Just a thought 😅

File: site/en/tutorials/eager/_pix2pix.ipynb (hunk 1 above)
[the remark this answers is not in the document]
Comment (lamberta) [reply site/en/tutorials/eager/_pix2pix.ipynb]: Hey Yash, I think we should just leave it as-is with the SVG. We can cross that road if we ever need to translate the diagram. Thanks

**Comment by lamberta** [comment]
SVG files (from Yash):
Generator (https://www.dropbox.com/s/f9d6drx1sczeonx/gen.svg?dl=0)
Discriminator (https://www.dropbox.com/s/llfsvypih2gbx29/dis.svg?dl=0)

**Comment by yashk2810** [comment]
Fixing this: https://github.com/tensorflow/tensorflow/issues/23432

**Comment by yashk2810** [comment]
@MarkDaoust @lamberta Is anything else needed from my side?

**Comment by lamberta** [comment]
Hi Yash, sorry for the delay. There are a bunch of notebook collections planned so we're still straightening things out.
Can you move this to the `site/en/tutorials/images` directory (and keep the underscore at the beginning of the name).
After that, we're good

**Comment by yashk2810** [comment]
> Hi Yash, sorry for the delay

No problem. I understand :smiley: 

> Can you move this to the site/en/tutorials/images directory (and keep the underscore at the beginning of the name).

Done.

File: site/en/tutorials/images/_pix2pix.ipynb [hunk 2]
```diff
@@ -1,31 +1,49 @@
...
-        "    \u003cimg src=\"https://www.tensorflow.org/images/colab_logo_32px.png\" /\u003eRun in Google Colab\u003c/a\u003e  \n",
-        "\u003c/td\u003e\u003ctd\u003e\n",
-        "\u003ca target=\"_blank\"  href=\"https://github.com/tensorflow/tensorflow/tree/master/tensorflow/contrib/eager/python/examples/pix2pix/pix2pix_eager.ipynb\"\u003e\u003cimg width=32px src=\"https://www.tensorflow.org/images/GitHub-Mark-32px.png\" /\u003eView source on GitHub\u003c/a\u003e\u003c/td\u003e\u003c/table\u003e"
+        "<table class=\"tfo-notebook-buttons\" align=\"left\"><td>\n",
+        "<a target=\"_blank\"  href=\"https://colab.research.google.com/github/tensorflow/tensorflow/blob/master/tensorflow/contrib/eager/python/examples/pix2pix/pix2pix_eager.ipynb\">\n",
+        "    <img src=\"https://www.tensorflow.org/images/colab_logo_32px.png\" />Run in Google Colab</a>  \n",
+        "</td><td>\n",
+        "<a target=\"_blank\"  href=\"https://github.com/tensorflow/tensorflow/tree/master/tensorflow/contrib/eager/python/examples/pix2pix/pix2pix_eager.ipynb\"><img width=32px src=\"https://www.tensorflow.org/images/GitHub-Mark-32px.png\" />View source on GitHub</a></td></table>"
```
Comment (lamberta) [inline site/en/tutorials/images/_pix2pix.ipynb]: https://github.com/tensorflow/docs/blob/master/site/en/tutorials/images/_pix2pix.ipynb

File: site/en/tutorials/images/_pix2pix.ipynb [hunk 3]
```diff
@@ -1,31 +1,49 @@
...
         "\n",
         "# Pix2Pix: An example with tf.keras and eager\n",
         "\n",
-        "\u003ctable class=\"tfo-notebook-buttons\" align=\"left\"\u003e\u003ctd\u003e\n",
-        "\u003ca target=\"_blank\"  href=\"https://colab.research.google.com/github/tensorflow/tensorflow/blob/master/tensorflow/contrib/eager/python/examples/pix2pix/pix2pix_eager.ipynb\"\u003e\n",
-        "    \u003cimg src=\"https://www.tensorflow.org/images/colab_logo_32px.png\" /\u003eRun in Google Colab\u003c/a\u003e  \n",
-        "\u003c/td\u003e\u003ctd\u003e\n",
-        "\u003ca target=\"_blank\"  href=\"https://github.com/tensorflow/tensorflow/tree/master/tensorflow/contrib/eager/python/examples/pix2pix/pix2pix_eager.ipynb\"\u003e\u003cimg width=32px src=\"https://www.tensorflow.org/images/GitHub-Mark-32px.png\" /\u003eView source on GitHub\u003c/a\u003e\u003c/td\u003e\u003c/table\u003e"
+        "<table class=\"tfo-notebook-buttons\" align=\"left\"><td>\n",
+        "<a target=\"_blank\"  href=\"https://colab.research.google.com/github/tensorflow/tensorflow/blob/master/tensorflow/contrib/eager/python/examples/pix2pix/pix2pix_eager.ipynb\">\n",
```
Comment (lamberta) [inline site/en/tutorials/images/_pix2pix.ipynb]: https://colab.research.google.com/github/tensorflow/docs/blob/master/site/en/tutorials/images/_pix2pix.ipynb

File: site/en/tutorials/images/_pix2pix.ipynb [hunk 4]
```diff
@@ -608,6 +626,15 @@
         "* The discriminator receives the input_image and the generated image as the first input. The second input is the input_image and the target_image.\n",
         "* Next, we calculate the generator and the discriminator loss.\n",
         "* Then, we calculate the gradients of loss with respect to both the generator and the discriminator variables(inputs) and apply those to the optimizer.\n",
+        "* This entire procedure is shown in the images below.\n",
+        "\n",
+        "![alt text](https://raw.githubusercontent.com/yashk2810/yashk2810.github.io/master/dis.jpg)\n",
```
Comment (lamberta) [inline site/en/tutorials/images/_pix2pix.ipynb]: Replace "alt text".

Yes, add an ./images directory to put these instead of your personal account (similar to the text_generation notebook in [sequences/](https://github.com/tensorflow/docs/tree/master/site/en/tutorials/sequences)).

File: site/en/tutorials/images/_pix2pix.ipynb [hunk 5]
```diff
@@ -1,31 +1,49 @@
...
     },
     {
-      "cell_type": "markdown",
       "metadata": {
         "colab_type": "text",
         "id": "ITZuApL56Mny"
       },
+      "cell_type": "markdown",
       "source": [
         "This notebook demonstrates image to image translation using conditional GAN's, as described in [Image-to-Image Translation with Conditional Adversarial Networks](https://arxiv.org/abs/1611.07004). Using this technique we can colorize black and white photos, convert google maps to google earth, etc. Here, we convert building facades to real buildings. We use [tf.keras](https://www.tensorflow.org/programmers_guide/keras) and [eager execution](https://www.tensorflow.org/programmers_guide/eager) to achieve this.\n",
```
Comment (lamberta) [inline site/en/tutorials/images/_pix2pix.ipynb]: Update URLs for:
https://www.tensorflow.org/guide/keras
https://www.tensorflow.org/guide/eager

Comment (yashk2810) [reply site/en/tutorials/images/_pix2pix.ipynb]: Done.

File: site/en/tutorials/images/_pix2pix.ipynb (hunk 4 above)
Comment (yashk2810) [reply site/en/tutorials/images/_pix2pix.ipynb]: Done.

File: site/en/tutorials/images/_pix2pix.ipynb (hunk 3 above)
Comment (yashk2810) [reply site/en/tutorials/images/_pix2pix.ipynb]: Done.

File: site/en/tutorials/images/_pix2pix.ipynb (hunk 2 above)
Comment (yashk2810) [reply site/en/tutorials/images/_pix2pix.ipynb]: Done.

**Comment by yashk2810** [comment]
@lamberta I have added the .svg and .jpg files in ./images.
